bundle/porter.yaml

```
@@ -59,15 +59,25 @@ parameters:
     default: vault-7f000001.nip.io
 
 outputs:
-  - name: kubeconfig
+  - name: kubeconfig-internal
     type: file
     path: /home/nonroot/.kube/config
     applyTo:
       - install
+  - name: kubeconfig
+    type: file
+    path: /home/nonroot/.kube/config-external
```

I think it would make sense to match the suffix used on the file to the output name.
```suggestion
  - name: kubeconfig
    type: file
    path: /home/nonroot/.kube/config
    applyTo:
      - install
  - name: kubeconfig-external
    type: file
    path: /home/nonroot/.kube/config-external
```